index.html

```diff
@@ -27,16 +27,54 @@
 
     <!-- section ABOUT begin -->
     <section class="container">
-        <div class="row">
-            ABOUT
+        <div class="row triple">
```

Comment:
gal tiesiog left clasė vietoj triple, nes labai maišosi ir tikiesi 3 blokų ar elementų, bet realiai tai tik paveiksliukų blokas.

Reply:
šis jei teisingai suskaičiavau būtų paveiksliukų bendras blokas, o triple turėtų apimti paveiksliukus ir dešinį bloką. Gal vitoj triple būtų papraščiai virsutis (top) ? atviras klausimas...

Reply:
triple gridas cia

Reply:
o left galima bus panaikinti, bet nujauciu, kad stylint galima bus leftus is skirtingu sekciju kartu

Reply:
pavadinti viršutiniu netiktų, kitaip iš pat pradžių pamačiau

Reply:
paveiksliukų bendras bloko plotis didesnis nei tekstinio bloko

Reply:
Tai as siulau kam nors kitam tada padaryt sitas uzduotis, as nesuprantu kaip daryt tada:)

Reply:
tai čia viskas manau gerai, tik tavo triple yra pavadintas paveiksliukų blokas, tai tą "triple" apimk kitu bloku ir jį pavadink triple

Reply:
aš tik galvoju, kad iš pat pradžių reikia gerai sužiūrėti, kad vėliau nebūtų per sunku viską pataisyti detales

Reply:
Is esmes sutinku, bet jei listi dbr iskarto gilyn i detales, galim pasimest. Manau kaip pradinis variantas geras.